MauiBundler/Plugins/Plugin.Filesystem/Services/FilesystemService.android.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.JSInterop;
+
+namespace Plugin.Filesystem.Services;
+
+
+public class FilesystemService : IFilesystemService
+{
+    [JSInvokable("deleteFile")]
+    public async ValueTask<bool> DeleteFile(string path)
+    {
+        var docsDirectory = Android.App.Application.Context.GetExternalFilesDir(Android.OS.Environment.DirectoryDocuments);
```

Comment:
Can we set this as a property?

Reply:
There will be more than one directory. 
We need to add a enum for use specific directories. 
Will be done later

Reply:
okay